contract/src/api.rs

```diff
@@ -176,10 +184,25 @@ impl Contract {
             // First key receives all the deposit.
             attached_balance = 0;
         }
+
         if attached_balance > 0 {
             env::panic_str("The attached deposit could not be added to any account");
         }
-        SetReturnType { block_height: U64(near_sdk::env::block_height()) }
+
+        if options.callback_receiver_id.is_some() {
+            let callback_receiver_id = options.callback_receiver_id.unwrap();
+
+            callback_contract::on_social_set(
+                near_sdk::env::block_height(),
+                callback_receiver_id.clone(),
+                0,
+                Gas(5 * TGAS),
```

Comment:
Let's let users specify the `callback_attached_gas` (optional; default to 5 TGas)